cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,28 @@
+import { generateUser } from '../support/generate';
+
+Cypress.on('uncaught:exception', () => {
+  return false;
+});
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const { email, username, password } = generateUser();
+
+    cy.visit('/#/register?_k=spysa9');
+
+    cy.get('[placeholder=Username]')
+      .type(username);
+
+    cy.get('[placeholder=Email]')
+      .type(email);
+
+    cy.get('[placeholder=Password]')
+      .type(password);
+
+    cy.get('button[type=submit]').click();
+
+    cy.url().should('contain', '/#/?_k=c1q9kl');
 
+    cy.get('a.nav-link').contains(username).should('exist');
```

Comment:
It's better to use more specific selectors than 'a.nav-link'. This will make your tests more resilient to changes in the HTML structure. Consider adding a data-test attribute to the element you want to select.